### ISSUE TITLE
License collision

### ISSUE THREAD

**Issue by jouniaro** [wadesalazar/NIFI-OPCUA#10]
The OPC Foundation Java stack is distributed under GPL. So, unfortunately, you cannot provide Nifi under Apache license. Your only option is GPL.

**Comment by wadesalazar** [comment]
I am providing only the nifi processor code here under the apache license.  Users are instructed to download, compile and install the stack separately.

**Comment by jouniaro** [comment]
Yes, but since your project depends on the OPC Foundation Java Stack, the GPL flows to your code as well. All applications built with your library must be GPL, because of this anyway. So, even if it would be technically possible (which I am unsure of), you are giving a false signal for the users of your library.

https://www.gnu.org/licenses/old-licenses/gpl-2.0-faq.html#IfLibraryIsGPL

**Comment by jouniaro** [comment]
On the other hand, of course, if the library is used by an OPC Foundation member company, they can then apply the RCL license to the OPC UA Java Stack, in which case Apache license here is a better alternative. Anyway, I think it would be good, if you would mention the licensing issues involved to your users as well.
